nodeJS/authentication/authentication_basics.md

```diff
@@ -90,7 +90,7 @@ Our view engine is set up to just look in the main directory, and it's looking f
 
 ### Creating users
 
-The first thing we need is a sign up form so we can actually create users to authenticate! In the Library Tutorial website, you learned about validating and sanitizing inputs. This is a *really good idea*, but for the sake of brevity, we're going to leave that out here. Don't forget to include sanitation and validation when you get to the project.
+The first thing we need is a sign up form so we can actually create users to authenticate! For the sake of brevity, we're going to leave sanitization and validation out here. But don't forget about it when you get to that point, we will have the opportunity to apply them later.
```

Comment:
Sanitization is the correct term I believe. And library project is not actual anymore